### PR TITLE
Action cable gf

### PR DESCRIPTION
**Pull request by gregoryjflaherty**
## Features
1. Create messages and social table 
2. Create channel for social board
3. Create new schema 
4. Create new message action  

## Concerns
1. Need to make sure we are able to connect the cable server to the FE 

## Pre-Submittal Checklist
- [X] All tests pass on local
- [X] Visually confirm view functionality
- [X] No prys or save_and_open_page commands remain
- [X] Unnecessary commented code removed

## As a GIF, how does this PR make you feel?

unagi

### CODE REVIEW

File: app/controllers/api/v1/social_controller.rb [hunk 1]
```diff
@@ -2,12 +2,21 @@ class Api::V1::SocialController < ApplicationController
...
+    user = User.find_by(name: params[:name])
+    channel = Social.first
+    message = Message.create!(
+      content: params[:content],
+      name: user.name,
+      social_id: channel.id,
+      user_id: user.id
+    )
+
+    SocialChannel.broadcast_to(channel, message)
```
Comment (samlsmith424) [inline app/controllers/api/v1/social_controller.rb:18]: Great formatting of the broadcasting the desired information.

File: app/controllers/api/v1/social_controller.rb [hunk 2]
```diff
@@ -2,12 +2,21 @@ class Api::V1::SocialController < ApplicationController
...
     messages = Message.all
-
     render json: messages
   end
 
   def create
-    
+    user = User.find_by(name: params[:name])
+    channel = Social.first
+    message = Message.create!(
```
Comment (samlsmith424) [inline app/controllers/api/v1/social_controller.rb:11]: I like how you broke this down for readability.

File: app/channels/social_channel.rb [hunk 3]
```diff
@@ -0,0 +1,11 @@
+class SocialChannel < ApplicationCable::Channel
+  def subscribed
```
Comment (samlsmith424) [inline app/channels/social_channel.rb:2]: Fantastic method in going about the group social channel.

File: db/migrate/20220525161203_add_socials_to_messages.rb [hunk 4]
```diff
@@ -0,0 +1,5 @@
+class AddSocialsToMessages < ActiveRecord::Migration[5.2]
```
Comment (SullyBirashk) [inline db/migrate/20220525161203_add_socials_to_messages.rb:1]: Hopefully this is all we need to get WebSockets live!

**Pull request merged**